docs/source/faq.md

```diff
@@ -351,6 +351,13 @@ In some cases, the internal memory type cache can misdetect GPU memory as host
 memory, also leading to invalid memory access. This cache can be disabled by
 setting `UCX_MEMTYPE_CACHE=n`.
 
+#### Seeing the error "provided PTX was compiled with an unsupported toolchain"?
+
+The name of the class of NVIDIA 'virtual' GPU architecture for which CUDA input
+files were compiled does not match with underlying GPU architecture. Refer to
+https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#virtual-architecture-feature-list
+for the appropriate `-arch` option to pass to nvcc.
```

Comment:
this is very unclear...
i'd write something like "The application is loading a cuda binary that was compiled for newer version of cuda than installed, and the failure is detected asynchronously by a Cuda API call from UCX. In order to fix the issue, need to install a newer cuda version or compile the cuda binaries with the appropriate -arch option to nvcc"